.github/scripts/fetch_prs.mjs

```diff
@@ -51,56 +56,126 @@ async function fetchReviewerDetails(login) {
   }
 }
 
-async function formatPR(pr) {
+async function fetchReviewsForPR(prNumber) {
+  try {
+    const { data } = await octokit.request('GET /repos/{owner}/{repo}/pulls/{pull_number}/reviews', {
+      owner,
+      repo,
+      pull_number: prNumber,
+    });
+    return data;
+  } catch (error) {
+    console.error(`Error fetching reviews for PR ${prNumber}:`, error);
+    return [];
+  }
+}
+
+async function formatPR(pr, isReadyForMerge) {
+// async function formatPR(pr) {
   let reviewersInfo = `NO REVIEWERS! ${defaultLogin}, your care is required here.`;
+  const fetchedReviewers = [];
+  
+  // Add requested reviewers
   if (pr.requested_reviewers && pr.requested_reviewers.length > 0) {
-    const reviewers = await Promise.all(
-      pr.requested_reviewers.map(async reviewer => {
-        const details = await fetchReviewerDetails(reviewer.login);
-        return details ? `${details.login} (${details.name})` : reviewer.login;
-      })
-    );
-    reviewersInfo = `Reviewers: ${reviewers.join(', ')}`;
+    for (const reviewer of pr.requested_reviewers) {
+      const details = await fetchReviewerDetails(reviewer.login);
+      const rewName = details.name ? details.name.replace(/ /g, '.').toLowerCase() : 'No name';
+      fetchedReviewers.push(details ? `${details.login} (@${rewName})` : reviewer.login);
+    }
+  }
+
+  // Add reviewers from reviews
+  const reviews = await fetchReviewsForPR(pr.number);
+  const uniqueReviewers = new Set(fetchedReviewers); // To avoid duplicates
```

Comment:
You can use logins for this this set to reduce formatting dublicates: `${details.login} (@${rewName})`

Reply:
```

function formatReviewer(reviewer, details) {
  if (!details) {
    return reviewer.login;  
  }
  const loopName = details.name ? details.name.replace(/ /g, '.').toLowerCase() : 'Set name in profile!';
  return `${details.login} (@${loopName})`;
}

async function formatPR(pr) {
...
  const uniqueLogins = new Set();
  const fetchedReviewers = [];

  // Add requested reviewers
  if (pr.requested_reviewers && pr.requested_reviewers.length > 0) {
    for (const reviewer of pr.requested_reviewers) {
       uniqueLogins.add(details.login);
       const details = await fetchReviewerDetails(reviewer.login);
       fetchedReviewers.push(formatReviewer(reviewer, details));
    }
  }

  // Add more reviewers from reviews.
  const reviews = await fetchReviewsForPR(pr.number);
  for (const review of reviews) {
    if (uniqueLogins.has(review.user.login)) {
      continue;
    }
    uniqueLogins.add(review.user.login);
    const details = await fetchReviewerDetails(review.user.login);
    fetchedReviewers.push(formatReviewer(reviewer, details));
  }
```

Reply:
Fixed